Indexer.php

```diff
@@ -71,6 +76,30 @@ public function isUrlIndexedandNotExpired($uri, array $metadata = [])
     }
 
     /**
+     * Returns url's that are expired.
+     *
+     * @param string $core
+     *
+     * @return \Solarium\QueryType\Select\Result\Result
+     */
+    public function findExpiredUrls($core)
+    {
+        $this->client->getEndpoint()->setCore($core);
+
+        $expiresBeforeDate = new DateTime();
+        $expiresBeforeDate->modify(sprintf('-%s hour', $this->hoursBeforeExpired));
+
+        $queryPhrase = sprintf("updatedDate:[* TO %s]", $expiresBeforeDate->format('Y-m-d\TH:i:s\Z'));
+
+        $query = $this->client->createSelect();
+        $query->setQuery($queryPhrase);
+
+        $result = $this->client->select($query);
+
+        return $result;
```

Comment:
is not better to have "return $this->client->select($query);", or you plan to do something with result